tutorindigo/templates/indigo/lms/templates/index.html

```diff
@@ -11,6 +11,8 @@
 <main id="main" aria-label="Content" tabindex="-1">
     <section class="home style-logout">
       <header>
+        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet">
```

Comment:
I would like to understand the reasoning behind this addition,  is it necessary?, from a user perspective could mean loading more files at the first content rendering.

Reply:
Removed! Thanks <3